socorro/external/crashstorage_base.py

```diff
@@ -620,7 +580,6 @@ def save_processed(self, processed_crash):
         if storage_exception.has_exceptions():
             raise storage_exception
 
-    #--------------------------------------------------------------------------
```

Comment:
I think if we don't like these, we should do a one-time pass and remove them all. I'm game for doing that if we want to go that route. Seems like an easy pass to do towards cleaning things up. I'd rather do that (spend an hour to do it once and for all) than have it show up in PRs (have to skim through them every time I review a PR).